src/components/data/listing/TableView.js

```diff
@@ -87,7 +87,7 @@ function TableView(props) {
     const tableColumns = getTableColumns(isMedium, isLarge);
 
     return (
-        <TableContainer component={Paper}>
+        <TableContainer component={Paper} style={{minHeight: 480, maxHeight: isMedium ? 1024 : 600}}>
```

Comment:
🤔 Do we want a max height?

Reply:
I don't think scrollbars will show up without maxHeight

Reply:
I don't know if it's related to the maxHeight, but I can't see the footer on the table when the full listing is > 1 page.